guava/src/com/google/common/escape/Escapers.java

```diff
@@ -211,14 +216,31 @@ public static String computeReplacement(CharEscaper escaper, char c) {
    * @param cp the Unicode code point to escape if necessary
    * @return the replacement string, or {@code null} if no escaping was needed
    */
-  public static String computeReplacement(UnicodeEscaper escaper, int cp) {
+  public static String computeReplacement(UnicodeEscaper escaper, @NonNegative int cp) {
     return stringOrNull(escaper.escape(cp));
   }
 
   private static String stringOrNull(char[] in) {
     return (in == null) ? null : new String(in);
   }
 
+  @SuppressWarnings("array.access.unsafe.high")/*
+  (262, 20) and (263, 18)
```

Comment:
These numbers are meaningless without more context. You should specify what they are. Or, better, remove them and add comments or annotations actually at the lines that they are referring to, since any source code change in the future would invalidate them.